ts/src/flexible-event/privacy.ts

```diff
@@ -90,14 +93,21 @@ export class Config {
 
   computeConfigData(epsilon: number, infoGainMax: number): ConfigData {
     const numStates = this.numFlexibleStates()
-    const infoGain = maxInformationGain(numStates, epsilon)
+    const infoGain = maxInformationGain(
+      numStates,
+      epsilon,
+      this.attributionScopeLimit,
+      this.maxEventStates
+    )
     const flipProb = flipProbabilityDp(numStates, epsilon)
 
     const data: ConfigData = { numStates, infoGain, flipProb }
 
     if (infoGain > infoGainMax) {
       const newEps = epsilonToBoundInfoGainAndDp(
         numStates,
+        this.attributionScopeLimit ?? 1,
+        this.maxEventStates ?? constants.defaultMaxEventStates,
```

Comment:
The right-hand side of these two `??` operations is never used, as `this.attributionScopeLimit` and `this.maxEventStates` are never null or undefined.

Reply:
Nice catch.